internal/main.go

```diff
@@ -22,9 +22,7 @@ import (
 	"os"
 	"strings"
 
-	"github.com/coreos/ign2to3"
```

Comment:
It looks like there are things that are still using stuff in the `ign2to3` package in the file even after this change.

I'm not sure what the correct answer is to import it though (maybe `github.com/coreos/ign-converter/ign2to3` or just `github.com/coreos/ign-converter`?)

Reply:
gahhh let me fix again.. (I swear i did this already)

Reply:
ok done